enacts/flex_fcst/cpt.py

```diff
@@ -106,17 +106,35 @@ def starts_list(
     return start_dates
 
 
-def read_pycptv2dataset(data_path):
+def read_mpycptv2dataset(data_path, SL_dense=True):
```

Comment:
Instead of passing this SL_dense value around, can't we just infer that SL_dense is true when the data has an L dimension, and false when it doesn't?

Reply:
read_mpycptv2dataset creates the L dimension so from there down the road in the maprooms, we could check whether the data has an L and choose what to do. Or if you have some other workflow in mind, let me know.

Reply:
@aaron-kaplan , as things stand, and with changes to the config file for which there is a PR in bitbucket too, The app works well with 2 sets of "SL-dense" forecasts (I copied the 2 forecasts Peterson has in their server). So they could have that for their workshop starting Tuesday. I am not saying we should merge as is, but we have little time before that workshop, and I also haven't started to look at IRI Strategy dcument that I would like to make sure I have time to look at today.

So I could do some important but not time-consuming here before merging, and we could put the rest as Issues.

Reply:
Replied on the config PR https://bitbucket.org/iridl/python_maproom_kmd/pull-requests/7/new-config-system-to-cover-sl_dense-true?link_source=email#comment-443730056

Reply:
ok. I fixed the 2ndary problems and now I am going to see what I can do free the config file from the boolean SL_dense.

Reply:
I pushed something to that effect. Tested on _kmd and _lesotho for which I also pushed the suppression of SL_dense from the config.